src/views/css/packlink-core-override.css

```diff
@@ -0,0 +1,17 @@
+#pl-page .pl-form-group input,
+#pl-page .pl-form-group select {
+  height: 100%;
+}
+
+#pl-pricing-policy-form .pl-form-group {
+  height: 100%
+}
+
+.pl-config-list-item svg {
+  width: 22px;
+  height: 22px;
+}
+
+.pl-half-width .material-icons {
+  padding-top: 5px !important;
```

Comment:
This should be reviewed. There should not be a need for such override. If there is, pudating core styles should be considered.

Reply:
I thought that this was a module-specific style issue, but after checking the DemoUI and WooCommerce integration, I noticed that the select boxes on the order status mapping page are displayed the same way. To me, it looks like the material icons for the drop-down list in select boxes are a bit higher than they should be. But if that has passed the review in Core and the other integration, I will remove this Core override file entirely since it is no longer required.